packages/block-editor/src/components/list-view/index.js

```diff
@@ -71,11 +73,20 @@ function ListViewComponent(
 		isExpanded = false,
 		showAppender = false,
 		blockSettingsMenu: BlockSettingsMenu = BlockSettingsDropdown,
+		rootClientId = null,
```

Comment:
```suggestion
		rootClientId,
```
I think it's ok to leave this undefined, or copy what `__unstableGetClientIdsTree` does and have an empty string as the default.